Fenced_Frames_Ads_Reporting.md

```diff
@@ -164,7 +164,8 @@ Since the `reserved.top_navigation` beacons are automatically generated by the b
 window.fence.setReportEventDataForAutomaticBeacons({
   'eventType': 'reserved.top_navigation',
   'eventData': 'an example string',
-  'destination':['seller', 'buyer']
+  'destination': ['seller', 'buyer'],
+  'once': false
```

Comment:
This line strikes me as a bit confusing:
1. it's well before we've introduced the "once" concept/option.
2. it's set to false which doesn't have an effect

WDYT about removing this change and adding an example below just before or after the "once" option is introduced/explained?

Reply:
sgtm - I'll update it.